Composer/packages/client/src/pages/publish/PublishStatusList.tsx

```diff
@@ -160,7 +162,7 @@ export const PublishStatusList: React.FC<IStatusListProps> = (props) => {
             allowDisabledFocus
             styles={{ root: { color: '#0078D4' } }}
```

Comment:
Check matching color in SharedColors or NeutralColors. It will be useful when we add themes to Composer